packages/suite/src/components/firmware/FirmwareOffer.tsx

```diff
@@ -101,7 +108,12 @@ const FirmwareOffer = ({ device, customFirmware }: Props) => {
                         <Label>
                             <Translation id="TR_ONBOARDING_CURRENT_VERSION" />
                         </Label>
-                        <Version>{currentVersion}</Version>
+                        <VersionWrapper>
+                            <Version>
+                                {currentVersion}
+                                {isBitcoinOnly(device) && ' (bitcoin-only)'}
```

Comment:
maybe good enough for now, but would be nice to make it translatable and maybe define it on one place and just use it on those three places

Reply:
I was thinking about making it translatable, but it is the name of the fw https://github.com/trezor/webwallet-data/tree/master/firmware/1 so maybe it would be confusing for users. `(bitcoin-pouze), (bitcoin-nur),...`

True, I had it like that before but I changed it back and I do not remember why 😄  I will change it.

Reply:
(bitcoin-only) is used in more places so I skipped it. 
Otherwise fixed.

Reply:
ok, let's wait till (if) product will ask to edit